### PR TITLE
Change quality #1006

### PR DESCRIPTION
**Pull request by RaynardGerraldo**
# Pull Request Template

## Type of change

- [ ] Bug fix
- [X] Feature
- [ ] Documentation update

## Description

Hi bros, Change quality feature,everything working, cant confirm syncplay tho I dont have it

only 3 LOC :)))))

## Checklist

- [X] any anime playing
- [X] bumped version
---
- [X] next, prev and replay work
- [X] `-c` history and continue work
- [X] `-d` downloads work
- [ ] `-s` syncplay works
- [X] `-q` quality works
- [X] `-v` vlc works
- [X] `-e` select episode works
- [X] `-r` range selection works
- [X] `--dub` both work
- [X] all providers return links (not necessarily on a single anime, use debug mode to confirm)
---
- [X] `-h` help info is up to date
- [X] Readme is up to date
- [X] Man page is up to date

## Additional Testcases

- The safe bet: One Piece
- Episode 0: Saenai Heroine no Sodatekata ♭
- Unicode: Saenai Heroine no Sodatekata ♭
- Non-whole episodes: Tensei shitara slime datta ken (ep. 24.5, ep. 24.9)

### CODE REVIEW

**Comment by port19x** [comment]
dang, thougt you were dead or sth

File: ani-cli [hunk 1]
```diff
@@ -376,16 +376,19 @@ tput cuu1 && tput el
...
 	replay) ;;
 	next) ep_no=$(printf "%s" "$ep_list" | sed -n "/^${ep_no}$/{g;1!p;};h") 2>/dev/null ;;
 	select) ep_no=$(printf "%s" "$ep_list" | nth "Select episode: " "$multi_selection_flag") ;;
+	change_quality) episode=$(printf "%s" "$links" | sed -n '/^\([0-9]*p\)/p' | launcher)
+	episode=$(printf "%s" "$episode" | cut -d'>' -f2);;
 	*) exit 0 ;;
 	esac
 	[ -z "$ep_no" ] && die "Out of range"
 	play
+	episode=
```
Comment (port19x) [inline ani-cli:391]: this looks a little sus

Comment (port19x) [inline ani-cli:391]: this looks a bit sus

**Review by port19x — APPROVED** [review]

File: ani-cli (hunk 1 above)
Comment (RaynardGerraldo) [reply ani-cli:391]: dont worry its to adjust with 214, selecting quality in the menu just uses already scraped link so when you press next without emptying episode it'll just replay the same episode again and again

tested everything as well , should be all fine

**Comment by RaynardGerraldo** [comment]
> dang, thougt you were dead or sth

2022 was crazy for me man, pulled through tho 👍

**Comment by RaynardGerraldo** [comment]
> `feat: change quality interactively`

im forgetting things here, this is done on the merge right?

**Comment by 71zenith** [comment]
the change isnt persistent, it will again use the default for other episodes

**Comment by 71zenith** [comment]
> > `feat: change quality interactively`
> 
> im forgetting things here, this is done on the merge right?
we squash the commits on merge to a single commit
but semantic commits are still needed

**Comment by RaynardGerraldo** [comment]
> the change isnt persistent, it will again use the default for other episodes

Yes ,users would just use -q if they wanted this tho right? First thing that I thought of this feature was like this is for those moments when ur 2 mins in the show and its lagging so you change the resolution

Even in a situation like that you could still just use -q but if you're not trying to be fully handsfree like doing a marathon and just trying to get 1 episode done might as well just use this feature

Either way its a pretty weird feature to be added again but thought it was just 3 LOC so why not,unless it causes more trouble

**Comment by RaynardGerraldo** [comment]
> the change isnt persistent, it will again use the default for other episodes

my bad its actually not complicated to keep it persistent

**Comment by RaynardGerraldo** [comment]
@port19x good to merge?

**Comment by candrapersada** [comment]
Add options to change the current providers after selecting ani-cli --dub koe no katachi
gambar
from  #1017

**Comment by port19x** [comment]
> Add options to change the current providers after run the command `ani-cli -d --dub koe no katachi` gambar

idk, seems sus

**Review by port19x — APPROVED** [review]

**Pull request merged**